homeassistant/components/tasmota/sensor.py

```diff
@@ -95,40 +111,66 @@
     hc.SENSOR_PB1: {ICON: "mdi:flask"},
     hc.SENSOR_PB2_5: {ICON: "mdi:flask"},
     hc.SENSOR_PB5: {ICON: "mdi:flask"},
-    hc.SENSOR_PM10: {ICON: "mdi:air-filter"},
-    hc.SENSOR_PM1: {ICON: "mdi:air-filter"},
-    hc.SENSOR_PM2_5: {ICON: "mdi:air-filter"},
-    hc.SENSOR_POWERFACTOR: {ICON: "mdi:alpha-f-circle-outline"},
-    hc.SENSOR_POWERUSAGE: {DEVICE_CLASS: DEVICE_CLASS_POWER},
+    hc.SENSOR_PM10: {
+        DEVICE_CLASS: SensorDeviceClass.PM10,
+        STATE_CLASS: SensorStateClass.MEASUREMENT,
+    },
+    hc.SENSOR_PM1: {
+        DEVICE_CLASS: SensorDeviceClass.PM1,
+        STATE_CLASS: SensorStateClass.MEASUREMENT,
+    },
+    hc.SENSOR_PM2_5: {
+        DEVICE_CLASS: SensorDeviceClass.PM25,
+        STATE_CLASS: SensorStateClass.MEASUREMENT,
+    },
+    hc.SENSOR_POWERFACTOR: {
+        ICON: "mdi:alpha-f-circle-outline",
+        STATE_CLASS: SensorStateClass.MEASUREMENT,
+    },
+    hc.SENSOR_POWERUSAGE: {
+        DEVICE_CLASS: SensorDeviceClass.POWER,
+        STATE_CLASS: SensorStateClass.MEASUREMENT,
+    },
     hc.SENSOR_PRESSURE: {
-        DEVICE_CLASS: DEVICE_CLASS_PRESSURE,
-        STATE_CLASS: STATE_CLASS_MEASUREMENT,
+        DEVICE_CLASS: SensorDeviceClass.PRESSURE,
+        STATE_CLASS: SensorStateClass.MEASUREMENT,
     },
     hc.SENSOR_PRESSUREATSEALEVEL: {
-        DEVICE_CLASS: DEVICE_CLASS_PRESSURE,
-        STATE_CLASS: STATE_CLASS_MEASUREMENT,
+        DEVICE_CLASS: SensorDeviceClass.PRESSURE,
+        STATE_CLASS: SensorStateClass.MEASUREMENT,
     },
     hc.SENSOR_PROXIMITY: {ICON: "mdi:ruler"},
-    hc.SENSOR_REACTIVE_POWERUSAGE: {DEVICE_CLASS: DEVICE_CLASS_POWER},
-    hc.SENSOR_STATUS_LAST_RESTART_TIME: {DEVICE_CLASS: DEVICE_CLASS_TIMESTAMP},
+    hc.SENSOR_REACTIVE_POWERUSAGE: {
+        STATE_CLASS: SensorStateClass.MEASUREMENT,
```

Comment:
```suggestion
        DEVICE_CLASS: SensorDeviceClass.POWER
        STATE_CLASS: SensorStateClass.MEASUREMENT,
```

Reply:
That's intentional, is not correct to set a sensor reporting reactive power as device class power